lib/getopt.ps1

```diff
@@ -25,9 +25,8 @@ function getopt($argv, $shortopts, $longopts) {
     }
 
     # ensure these are arrays
-    $argv = @($argv -split ' ')
+    $argv = @($argv -split ' \\(?=(?:[^"]|"[^"]*")*$)')
```

Comment:
```suggestion
    $argv = @($argv)
```

This will solve the problem...

Or change the function param line to `function getopt([String[]]$argv, [String]$shortopts, [String[]]$longopts)` and remove the two ensure lines.